baking.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Recipes and Relics: Baking</title>
+</head>
+<body>
+<header>
+    <nav><img src="images/Recipes_and_Relics.jpg" alt="Recipes and Relics Logo with Spoon" srcset=""></nav>
+    <ul>
+        <li><a href="cookery.html">Cookery</a></li>
+        <li><a href="baking.html">Baking</a></li>
+        <li><a href="drinks.html">Drinks</a></li>
+    </ul>
+</header>
+><main>>
```

Comment:
Hi @EmilyMorris10 you have a few validation errors. Please use the HTMLHint extension to view the problems.

Reply:
Thank you; I didn't realize I hadn't installed that yet because I still had the option of View>Problems. It should be fixed now.